hazelcast/src/main/java/com/hazelcast/spi/Operation.java

```diff
@@ -41,12 +41,19 @@
  */
 public abstract class Operation implements DataSerializable, RemotePropagatable<Operation> {
 
+    static final int BITMASK_VALIDATE_TARGET = 1;
+    static final int BITMASK_CALLER_UUID_SET = 2;
+    static final int BITMASK_REPLICA_INDEX_SET = 4;
+    static final int BITMASK_WAIT_TIMEOUT_SET = 8;
+    static final int BITMASK_PARTITION_ID_32_BIT = 16;
+    static final int BITMASK_CALL_TIMEOUT_64_BIT = 32;
+
     // serialized
     private String serviceName;
     private int partitionId = -1;
     private int replicaIndex;
     private long callId;
-    private boolean validateTarget = true;
+    private byte flags;
```

Comment:
What should we do with the executor? 

Should I do an optimal serialization for that one as well so the executor feature remains, even though it isn't used? Or should I leave it as it is for this PR and wait for the removal of the executor configuration in the executor to get merged?